style.css

```diff
@@ -33,29 +33,125 @@ h3,
 p {
   color: inherit;
 }
-img{
-  width:100%;
-  object-fit: contain;
+img {
+  width: 100%;
+  /* object-fit: contain; */
 }
 .grid-2 {
   display: grid;
   grid-template-columns: repeat(2, 1fr);
   gap: 2rem;
 }
-.homepage-body{
-  height:100vh;
-  display:grid;
-  grid-template-rows: auto 1fr 1fr;
-  grid-template-columns: auto 1fr;
-  grid-template-areas: 
-  "header header header"
-  "main   main   main"
-  "main   main   main";
-}
-.homepage-main{
-  grid-area:main;
-  overflow-y:auto;
-}
-.homepage-header{
-  grid-area:header;
+.grid-3 {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  gap: 2rem;
+}
+.checkbox-text{
+ margin:0 1rem;
+}
+/* header  */
+.arstyShop-header {
+  padding: 1rem 4rem;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100vw;
+  background: var(--secondary-color);
+}
+.arstyShop-header > * {
+  color: var(--light-text);
+}
+.right-side {
+  display: flex;
+  align-items: center;
+  gap: 2rem;
+  font-size: large;
+}
+.items {
+  color: var(--primary-color);
+  font-size: 1.4rem;
+}
+
+/* subscribe modal */
+.subscribe-section-overlay,
+.login-section-overlay,
+.signup-section-overlay {
```

Comment:
**Tip**: You can add a box shadow to darken the rest of the page when the modal is open. Try - ( box-shadow: 0 0 0 1600px rgba(0, 0, 0, 0.65);)
![Uploading Screenshot 2022-02-14 at 12.00.54 PM.png…]()